src/Data/InheritedFrom.elm

```diff
@@ -0,0 +1,15 @@
+module Data.InheritedFrom exposing
+    ( Model
+    , decode
+    )
+
+import Json.Decode as Decode exposing (Decoder)
+
+
+type Model
+    = Model (Maybe String)
```

Comment:
It is hard for me to undestand what the `Model` stands for. I think having it named InheritedFrom would be more readable.